tasks/main.yml

```diff
@@ -20,7 +20,7 @@
   tags:
     - haproxy
 
-- name: Start Marathon service
-  service: name=marathon state=restarted enabled=yes
+- name: Enable Marathon service
+  service: name=marathon enabled=yes
```

Comment:
@tsaridas to add a variable for this is overkill IMO
The only reasonable value choices:  'started' or 'omit'  (later being special and possibly not clear to the average ansible user)